### PR TITLE
add gem whenever

### PR DESCRIPTION
**Pull request by demands14**
## Related Tickets
- [#53573](https://edu-redmine.sun-asterisk.vn/issues/53573)

## WHAT (optional)
- Delete Expired Payment in schedule by gem whenever

## HOW
- Add gem when ever
- write rake task
- add schedule

## Evidence (Screenshot or Video)
image

### CODE REVIEW

**Comment by demands14** [comment]
Ready

File: lib/tasks/delete_expired_payments.rake [hunk 1]
```diff
@@ -0,0 +1,10 @@
+require "rake"
+namespace :delete_namespace do
+  desc "delete payments expired"
+  task delete_payments_expired: :environment do
+    @payments = Payment.all
```
Comment (huongnt-2545) [inline lib/tasks/delete_expired_payments.rake]: chỉ cần lấy ra những payment có trạng thái `in_active` để check thôi e nhé

File: config/schedule.rb [hunk 2]
```diff
@@ -0,0 +1,8 @@
+set :environment, "development"
+set :output, "log/cron_log.log"
+env :PATH, ENV["PATH"]
+
+#check_expiration of payments every 15 minutes
+every 15.minute do
```
Comment (huongnt-2545) [inline config/schedule.rb]: nên lặp mỗi 5p để check

**Comment by demands14** [comment]
Ready

**Review by huongnt-2545 — APPROVED** [review]
merged

**Pull request merged**